src/common/format.ts

```diff
@@ -39,6 +40,18 @@ export const formatDatetimeAsDate = (datetime: Date | string | null): string =>
     return new Date(datetime).toLocaleDateString("en-GB");
 };
 
+export const formatDatetimeAsTime = (datetime: Date | string | null): string => {
+    if (datetime instanceof Date) {
+        return datetime.toLocaleDateString("en-GB");
+    }
+
+    if (datetime === null || isNaN(Date.parse(datetime))) {
+        return "-";
+    }
+
+    return new Date(datetime).toLocaleTimeString("en-GB");
+};
```

Comment:
If it happens to be Date, does it break if you try to convert? i.e. just have 
```
if (datetime === null || isNaN(Date.parse(datetime))) {
    return "-";
}

return new Date(datetime).toLocaleTimeString("en-GB");
```

Reply:
I think Date.parse doesn't like it but I've removed Date as a parameter as its only used in the format parcel details bit